sdk/resourcegraph/Microsoft.Azure.Management.ResourceGraph/src/Generated/Models/QueryRequest.cs

```diff
@@ -32,15 +32,18 @@ public QueryRequest()
         /// <summary>
         /// Initializes a new instance of the QueryRequest class.
         /// </summary>
+        /// <param name="query">The resources query.</param>
         /// <param name="subscriptions">Azure subscriptions against which to
         /// execute the query.</param>
-        /// <param name="query">The resources query.</param>
+        /// <param name="managementGroupId">The management group
+        /// identifier.</param>
         /// <param name="options">The query evaluation options</param>
         /// <param name="facets">An array of facet requests to be computed
         /// against the query result.</param>
-        public QueryRequest(IList<string> subscriptions, string query, QueryRequestOptions options = default(QueryRequestOptions), IList<FacetRequest> facets = default(IList<FacetRequest>))
```

Comment:
breaking change.  You should add back the original constructor through customizations (since this is a partial class, adding a new partial class with the original constructor  that calls this one will prevent a breaking change).

Reply:
Thank you for the thourugh feedback! Apologize for not responding in a time manner, we've decided to go forward with the GA version, I'll post another PR and address your feedback there.